tests/pfc/test_pfc_pause_lossless.py

```diff
@@ -0,0 +1,352 @@
+            test_dscp_list = lossless_prio_dscp_map[prio]
+            # The other DSCP values.
+            bg_dscp_list = [x for x in range(64) if x not in test_dscp_list]
+
+            exp_dur = 2
+
+            send_pause_frame = False if send_pause_frame == True else True
+            paused = send_pause_frame
+            run_pfc_exp(session=session,
+                        dut=duthost,
+                        tx_port=tx_port,
+                        rx_port=rx_port,
+                        port_bw=tx_port_bw,
+                        test_prio_list=[prio],
+                        test_dscp_list=test_dscp_list,
+                        bg_dscp_list=bg_dscp_list,
+                        exp_dur=exp_dur,
+                        test_traffic_pause_expected=paused,
+                        send_pause_frame=send_pause_frame)
+
+            clean_configuration(session=session)
+
+
+def test_pfc_pause_multi_lossless_priorities(testbed, 
+                                             conn_graph_facts, 
+                                             lossless_prio_dscp_map,
+                                             duthost, 
+                                             ixia_dev, 
+                                             ixia_api_server_session,
+                                             fanout_graph_facts):
+    """
+    RDMA PFC - Pauses on multiple lossless priorities.
+    1. On SONiC DUT enable PFC on several priorities e.g priority 3 and 4.
+    2. Disable the PFC watchdog on the SONiC DUT.
+    3. On the Ixia Tx port create two flows - a) 'Test Data Traffic' and
+       b) 'Background Data traffic'.
+    4. Configure 'Test Data Traffic' such that it contains traffic items
+       with all lossless priorities.
+    5. Configure 'Background Data Traffic' it contains traffic items with
+       all lossy priorities.
+    6. From Rx port send pause frames on all lossless priorities. Then
+       start 'Test Data Traffic' and 'Background Data Traffic'.
+    7. When pause frames are started 'Test Data Traffic' will stop; 
+       and when pause frames are stopped 'Test Data Traffic' will start.
+    8. 'Background Data Traffic' will always flow.
+    9. Repeat the steps 4 to 8 several times.
+
+    """
+    port_list = list()
+    fanout_devices = IxiaFanoutManager(fanout_graph_facts)
+    fanout_devices.get_fanout_device_details(device_number=0)
+    device_conn = conn_graph_facts['device_conn']
+
+    for intf in fanout_devices.get_ports():
+        peer_port = intf['peer_port']
+        intf['speed'] = int(device_conn[peer_port]['speed'])
+        port_list.append(intf)
+
+    # The topology should have at least two interfaces.
+    pytest_assert(len(device_conn) >= 2,
+        "The topology should have at least two interfaces")
+
+    session = ixia_api_server_session
+    send_pause_frame = False
+    for i in range(len(port_list)):
+        vports = configure_ports(session, port_list)
```

Comment:
Add another loop 'for send_pause_frame in [True, False]':

Reply:
Done